### ISSUE TITLE
From zero to hundred km/h time

### ISSUE THREAD

**Issue by tampoldandre** [RP6conrad/ESP-GPS-Logger#19]
Hello! **10 Hz** is nice feature of the logger, thanks!

Is it possible to calculate time in device to measure 0 to 100 km/h?
Processing file, it is easy to find it, but would be nice to see on screen right after "run" also.
100+ km/h is often exceeded in winter https://www.gps-icesailing.com/ ..

Andre

**Comment by RP6conrad** [comment]
100 km/h can be logged already with the esp-gps as it is. The only issue is the e-paper screen layout, for font size for the speeds is only capable for display 99.9 km/h (3 digits).
In my car, I tested this several times, and the only issue is the layout of the screen.
So, the logger is certainly suited for icesailing !
To calculate acceleration from 0 km/h  to 100 km/h this can be done. But the accuracy would be less then 0.1s, as there are some delays involved, in the gps itself, and some in the logger.
This feature is for motor / car test ?

Greetings, Jan.

Verzonden vanuit Mail voor Windows

Van: tampoldandre
Verzonden: donderdag 20 april 2023 10:08
Aan: RP6conrad/ESP-GPS-Logger
CC: Subscribed
Onderwerp: [RP6conrad/ESP-GPS-Logger] From zero to hundred km/h time (Issue #19)

Hello! 10 Hz is nice feature of the logger, thanks!
Is it possible to calculate time in device to measure 0 to 100 km/h?
Processing file, it is easy to find it, but would be nice to see on screen right after "run" also.
100+ km/h is often exceeded in winter https://www.gps-icesailing.com/ ..
Andre

**Comment by tampoldandre** [comment]
Yes, max speed, max2, 10sec and avg over 100 km/h seems to work very well and will be measured next winter again on ice sailing.

The only thing I would like to see is calculated acceleration from 0 km/h  to 100 km/h. In summer yes, with motobike/car... I manually checked SBP file ESP GPS recorded and I can see that time from measured point speed less < 1 m/s (or 1 km/h) and then point where speed first time goes > 100km/h is ~4.2 seconds I would like to see on screen :) ... (0.1s/0.2s accuracy is very OK)

I mean: Screen choices if speed < 1 m/s (Statistics of session) and Screen choices if speed is > 1m/s to choose this calculated field...
